test/python/transpiler/test_collect_multiq_blocks.py

```diff
@@ -310,29 +310,25 @@ def test_collect_from_back(self):
         qc.cx(0, 3)
         qc.h(3)
 
+        dag = circuit_to_dag(qc)
+        # For the circuit above, the topological order is unique
+        topo_ops = list(dag.topological_op_nodes())
+
         # When collecting blocks of size-3 using the default direction,
         # the first block should contain the H-gate and two CX-gates,
         # and the second block should contain a single CX-gate and an H-gate.
-        pass_manager = PassManager()
-        pass_manager.append(CollectMultiQBlocks(max_block_size=3, collect_from_back=False))
-        pass_manager.run(qc)
-
-        block_list = pass_manager.property_set["block_list"]
-        self.assertEqual(len(block_list), 2)
-        self.assertEqual(len(block_list[0]), 3)
-        self.assertEqual(len(block_list[1]), 2)
+        pass_ = CollectMultiQBlocks(max_block_size=3, collect_from_back=False)
+        pass_.run(dag)
+        expected_blocks = [[topo_ops[0], topo_ops[1], topo_ops[2]], [topo_ops[3], topo_ops[4]]]
+        self.assertTrue(pass_.property_set["block_list"], expected_blocks)
```

Comment:
Presumably you mean `self.assertEqual` (and in the other place)?  This is asserting non-emptiness of the `pass_.property_set["block_list"]`, with a "message" field of `expected_blocks`.

Reply:
Oops, of course! Fixed in [65a1aba](https://github.com/Qiskit/qiskit/pull/13612/commits/65a1abad1e1a23509eca6612e683666e871c4d23).